game/external/RUNME.py

```diff
@@ -0,0 +1,18 @@
+import os
+import shutil
+import json
+
+manifest = open("manifest.txt", "r")
+files = manifest.read()
+manifest.close()
+files = json.loads(files)
+files.pop("example.file", "return")
+
+for file in files:
+    if not os.path.exists(file):
+        raise Exception("file '{0}' does not exist".format(file))
+    if not os.path.exists(files[file]):
+        raise Exception("directory '{0}' does not exist or isn't a valid directory".format(files[file]))
+    if os.path.exists("{0}/{1}".format(files[file], file)):
+        continue
+    shutil.move(file, files[file])
```

Comment:
Since this is a dict, we can iteritems
```suggestion
for file, target in files.iteritems():
    if not os.path.exists(file):
        raise Exception("File '{0}' does not exist".format(file))
    if not os.path.exists(target):
        raise Exception("Target directory '{0}' does not exist or isn't a valid directory".format(target))
    if os.path.exists("{0}/{1}".format(target, file)):
        continue
    shutil.move(file, files[file])
```